frontend/simple/js/backend/hapi.js

```diff
@@ -62,21 +62,14 @@ export class HapiBackend extends Backend {
   }
   publishLogEntry (contractId: string, entry: HashableEntry) {
     console.log(`publishLogEntry to ${contractId}:`, entry)
-    if (entry instanceof HashableAction) {
-      var contractType = store.state.contracts[contractId]
-      if (!Events[contractType].isActionAllowed(entry)) {
-        // TODO: post a proper notification to the user
-        console.error('entry not allowed', entry)
-        return Promise.reject('entry not allowed! this should never happen!')
-      }
-    }
+    // There used to be a permission check here buts its duplicated when a subcribed
+    // contract received the events
+    // in cases like send of messages the check information is not known so this check
+    // is better left to the server and the subscribers to perform the check
```

Comment:
This is fine... my thinking was why create potential problems for others if we can catch them on our side before sending them out? But I guess removing this will also help us to make the app more robust to failures we might not otherwise have caught. 👍